<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectDlg</class>
 <widget class="QDialog" name="NewProjectDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Project</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="6">
    <widget class="QToolButton" name="toolOverview">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QLineEdit" name="lineBase"/>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="pushOk">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>264</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Project title</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5" colspan="2">
    <widget class="QPushButton" name="pushCancel">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="QLineEdit" name="lineTitle"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineSuffix1"/>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Project base folder</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Suffixes</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="lineSuffix2"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineTitle</tabstop>
  <tabstop>lineBase</tabstop>
  <tabstop>lineSuffix1</tabstop>
  <tabstop>lineSuffix2</tabstop>
  <tabstop>toolOverview</tabstop>
  <tabstop>pushOk</tabstop>
  <tabstop>pushCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
